### PR TITLE
Add top level disputes

### PR DESCRIPTION
**Pull request by kyleconroy**
This is a breaking change and will require a new major version.

### CODE REVIEW

**Comment by kyleconroy** [comment]
r? @cosn cc @cupcait 

(no rush on this review, feel free to wait til Monday)

File: charge/client.go [hunk 1]
```diff
@@ -236,6 +236,43 @@ func (c Client) MarkSafe(id string) (*stripe.Charge, error) {
 		&stripe.ChargeParams{Fraud: ReportSafe})
 }
 
+func UpdateDispute(id string, params *stripe.DisputeParams) (*stripe.Dispute, error) {
```
Comment (cosn) [inline charge/client.go]: docs for public methods

File: dispute/client.go [hunk 2]
```diff
@@ -34,6 +34,76 @@ type Client struct {
...
+	}
+
+	dispute := &stripe.Dispute{}
+	err := c.B.Call("GET", "/disputes/"+id, c.Key, body, commonParams, dispute)
+
+	return dispute, err
+}
+
+// List returns a list of plans.
+// For more details see https://stripe.com/docs/api#list_plans.
```
Comment (cosn) [inline dispute/client.go]: c+p error: s/plans/disputes

File: dispute/client.go [hunk 3]
```diff
@@ -34,6 +34,76 @@ type Client struct {
...
+		for i, v := range list.Values {
+			ret[i] = v
+		}
+
+		return ret, list.ListMeta, err
+	})}
+}
+
+// Iter is an iterator for lists of Plans.
+// The embedded Iter carries methods with it;
```
Comment (cosn) [inline dispute/client.go]: ditto

**Comment by cosn** [comment]
:+1: 

we'll also need to add CHANGEDOC and bump `stripe.go:27`

**Comment by kyleconroy** [comment]
Alright, updated the documentation.

**Pull request merged**